### ISSUE TITLE
0.17.0-beta.17  jspm init causes a type error 'name' of undefined 

### ISSUE THREAD

**Issue by conor909** [jspm/jspm-cli#1889]
`jspm init`

After inserting my details I get this:

```
err  TypeError: Cannot read property 'name' of undefined
    at C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\lib\config\loader.js:431:61
    at Array.forEach (native)
    at JspmSystemConfig.Object.defineProperty.set (C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\lib\config\loader.js:429:41)
    at C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\lib\config\index.js:721:30
    at tryCatcher (C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\node_modules\bluebird\js\release\promise.js:502:31)
    at Promise._settlePromise (C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\node_modules\bluebird\js\release\promise.js:559:18)
    at Promise._settlePromise0 (C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\node_modules\bluebird\js\release\promise.js:604:10)
    at Promise._settlePromises (C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\node_modules\bluebird\js\release\promise.js:683:18)
    at Async._drainQueue (C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\node_modules\bluebird\js\release\async.js:138:16)
    at Async._drainQueues (C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\node_modules\bluebird\js\release\async.js:148:10)
    at Immediate.Async.drainQueues [as _onImmediate] (C:\Users\Conor\AppData\Roaming\npm\node_modules\jspm\node_modules\bluebird\js\release\async.js:17:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/jspm/jspm-cli/commit/a61c23c754b95351b141b53d8feee305e188f3f1.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.18.
